tomcat/src/main/java/org/apache/coyote/http/request/RequestLine.java

```diff
@@ -18,7 +18,10 @@ public static RequestLine of(String requestLine) {
         String originMethod = requestLines.get(0);
         String originPath = requestLines.get(1);
         String originVersion = requestLines.get(2);
-        return new RequestLine(HttpMethod.findByMethod(originMethod), new Path(originPath), HttpVersion.findByVersion(originVersion));
+        return new RequestLine(
```

Comment:
매번 클라이언트에서 RequestLine 객체를 만들 때마다 필요한 값을 직접 하나하나 설정하려니 불편하네요.
아래와 같이 파라미터를 넘겨주면 RequestLine 객체를 만드는 입장에서 더 편하지 않을까요?

```suggestion
        return new RequestLine(method, path, version);
```

Reply:
혹시 지금 방식을 의도하신게 맞는지 모르겠지만,
문자열로부터 필요한 객체를 찾아(혹은 만들어) 초기화 시키는 부분이라
생성자를 분리하였습니다.